content/3.Joining Corda Network/4.Onboarding Workflow.md

```diff
@@ -140,23 +156,37 @@ Start polling server for certificate signing approval.
 
 Important: the Request ID given in the above should be noted and kept safe for future reference.
```

Comment:
why? would be good if we can explain more clearly why they should keep the Request ID

Reply:
added more details to this